### PR TITLE
Added skip to main content link

### PR DESCRIPTION
**Pull request by Sayak9495**
This PR aims to solve #69 . @sharmi please review and let me know if any changes are required.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -21,3 +21,8 @@
   {% asset styles.css %}
   {% asset animate.css %}
 </head>
+
+<body>
```
Comment (astronomersiva) [inline _includes/head.html]: Please move this to the default and minimal layouts. This file should have only `head` related content.

**Review by astronomersiva — CHANGES_REQUESTED** [review]
Please move this to the default and minimal layouts. This file should have only head related content.

File: _includes/head.html (hunk 1 above)
Comment (astronomersiva) [inline _includes/head.html]: Please move this to the default and minimal layouts. This file should have only head related content.

Comment (Sayak9495) [reply _includes/head.html]: @astronomersiva pushed the requested changes. Please review and let me know.

**Comment by astronomersiva** [comment]
Looks good @Sayak9495! Thanks for this :)

Can you rebase your branch and resolve conflicts?

**Comment by Sayak9495** [comment]
@astronomersiva , I am having difficulty resolving the conflicts using rebase. I am a very basic git user and didn't used rebase before. It would be awesome if you or someone can help me out here.

**Comment by Sayak9495** [comment]
Opened a new PR hence closing this PR.
p.s sorry for the mess.

**Pull request closed without merging**